test/integration/controllers/message.spec.js

```diff
@@ -0,0 +1,67 @@
+const { serverInit, serverCleanup, stopServer } = require('~/test/setup')
+const testUserAuthentication = require('~/utils/testUserAuth')
+const { expectError } = require('~/test/helpers')
+const { UNAUTHORIZED } = require('~/consts/errors')
+
+const endpointUrl = '/messages/'
+// const nonExistingCategoryId = '63bed9ef260f18d04ab15da2'
```

Comment:
i don't see where it can be used so remove this commented line of code

Reply:
Cleaned.